### ISSUE TITLE
about the usecase of the project.

### ISSUE THREAD

**Issue by naren142** [NVlabs/SPADE#20]
Hi, can we generate the images of fake humans using this code?

**Comment by ArashHosseini** [comment]
+1

**Comment by taesungp** [comment]
COCO-stuff does contain human label, but it is actually a very hard problem to generate realistic human on datasets of human in various poses. The generated human images on COCO-stuff look pretty bad. 
The current best GAN models can only generate realistic human when [all the keypoints / part segmentations are given](https://arxiv.org/abs/1808.07371) , or when the human images are [aligned](https://github.com/NVlabs/stylegan). I think our model can be used to generate realistic human images if it is trained with part segmentation, but there is no pretrained model that does that.

**Comment by banyet1** [comment]
I've tried pix2pixHD(label and image) and other GAN to generate fake humans, the result is weird, most of them looks like just a mess of meats. Now I'm using stylegan (https://github.com/NVlabs/stylegan), maybe 
 it's the light of the tunnel, but it's take a lot of time for training (2x1080ti, 10K images, 10 day's passed, and it still wade on and on, and on).

**Comment by naren142** [comment]
can you post some of the example images which were generated using pix2pixHD?

**Comment by matthewbaykenney** [comment]
@naren142 here are a few I made w/ Pix2PixHD:
http://www.mattkenney.me/fake-news-with-openpose-and-pix2pix/
http://www.mattkenney.me/openpose-pix2pix/

**Comment by taki0112** [comment]
If you use the [CelebA-HQ Mask dataset](https://github.com/switchablenorms/CelebAMask-HQ). 
SPADE can also generate fake humans.

Here is my [tensorflow version result](https://github.com/taki0112/SPADE-Tensorflow)
